.changeset/witty-ducks-design.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sap-cloud-sdk/e2e-tests': patch
+'@sap-cloud-sdk/odata-common': patch
+'@sap-cloud-sdk/odata-v2': patch
+'@sap-cloud-sdk/odata-v4': patch
+---
+
+Deserializer load in batches with only function imports
```

Comment:
[req] Please rephrase this to a full sentence. I don't really get what is meant by this.